components/Table.tsx

```diff
@@ -183,7 +187,8 @@ function activeSynthsFilter(
   return rows.filter(
     (row) =>
       (row.original as Synth<{ type: ContractType }>).expirationTimestamp >
-      DateTime.now().toSeconds()
+        DateTime.now().toSeconds() ||
+      (row.original as ContentfulSynth).category === "External Integration"
```

Comment:
List synth on UMAverse if it is an External Integration or the contract is still active